recipes/whos-there/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "whos-there" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/whos-there-{{ version }}.tar.gz
+  sha256: a6378144b566b9bef4008b3a06e4720966514012577d6c97ad8cf38217763bdf
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
```

Comment:
Are we adding upper bounds now @conda-forge/core ?

Reply:
No, we should remove this. I've seen more of this lately. We may want to adjust the linter.

Reply:
If you use grayskull, it does automatically uses `python >=3.x,<4.0` at times.

Reply:
Removed the upper bound.